packages/flutter/lib/src/material/icon_button_theme.dart

```diff
@@ -55,7 +55,9 @@ class IconButtonThemeData with Diagnosticable {
     if (identical(a, b)) {
       return a;
     }
-    return IconButtonThemeData(style: ButtonStyle.lerp(a?.style, b?.style, t));
+    return IconButtonThemeData(
```

Comment:
This seems just reformatted the code and caused the `linux analyser` test failure. Reverting it back will fix the test.